combiner/lists/AM_SVG-full.lis

```diff
@@ -0,0 +1,14 @@
+jax/input/AsciiMath/config.js
+jax/output/SVG/config.js
+jax/output/FastHTML/config.js
+extensions/asciimath2jax.js
+extensions/MathEvents.js
+extensions/MathZoom.js
+extensions/MathMenu.js
+jax/element/mml/jax.js
+extensions/toMathML.js
```

Comment:
Sure, it's your PR, I'm happy for you to do it, but I can do it if you don't want to.